### PR TITLE
Make origin a ProxyResource instead of TrackedResource

### PR DESCRIPTION
**Pull request by nachakra**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by nachakra** [comment]
We have not released this API version as yet for public preview. It is scheduled for release next week. So, this is not a breaking change for the users.

**Comment by akning-ms** [comment]
> We have not released this API version as yet for public preview. It is scheduled for release next week. So, this is not a breaking change for the users.
Hi @nachakra,  just a confirm, understand it is not released yet,  is there any sdk released based on this api version?

**Comment by nachakra** [comment]
@akning-ms - No we have not released any SDK since our 2019-06-15-preview version of the API.

**Comment by akning-ms** [comment]
Hi @nachakra, Noticed you just have one more commit which also updated 2020-03-01 and 2020-04-15, plus 2019-12-31 totally 3 stable version.  so I am confused, are they all not released?  if yes, can I know why?  :)

File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json [hunk 1]
```diff
@@ -4332,7 +4332,7 @@
       "type": "object",
       "allOf": [
         {
-          "$ref": "#/definitions/TrackedResource"
+          "$ref": "#/definitions/ProxyResource"
```
Comment (akning-ms) [inline specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json:4335]: Can I know why have this change from business/service perspective? as it is a breaking change for a stable api version

Comment (nachakra) [reply specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json:4335]: It should have always been a proxy resource - it is associated with an endpoint which is a tracked resource. We don't need location/tags for this resource. As I said, this API version and the ones following it are not released as yet.

**Comment by nachakra** [comment]
> Hi @nachakra, Noticed you just have one more commit which also updated 2020-03-01 and 2020-04-15, plus 2019-12-31 totally 3 stable version. so I am confused, are they all not released? if yes, can I know why? :)

Because of covid related deployment freezes, the 2019-12-31 version slipped quite a bit. 2020-03-01 is in private preview but 2020-04-15 is the version(uber version with 2019-12-31 changes) that we are targeting to release since it is associated with JEDI requirements. So, we will probably skip 2019-12-31 version altogether if 2020-04-15 is ready to release next week.

**Review by akning-ms — APPROVED** [review]

**Comment by mikhailshilkov** [comment]
> Yes we have not released any new versions of SDKs since our 2019-06-15-preview version of the API. So, none of these API versions have any SDK released.

@nachakra Could you elaborate on what "released" means in this context?

Were the API endpoints available for these versions? If yes, this means that changes in specs are breaking?

For instance, `location` is still mentioned in ARM templates docs [here](https://docs.microsoft.com/en-us/azure/templates/microsoft.cdn/profiles/endpoints/origins). Was it possible to deploy ARM templates with these versions?

What is the place to track whether these or other spec versions are released or not?

**Pull request merged**